### ISSUE TITLE
Upgrade mongodb to 3.5.7 to fix warnings in Node.js 14

### ISSUE THREAD

**Issue by StarpTech** [seppevs/migrate-mongo#222]
https://jira.mongodb.org/browse/NODE-2536

**Comment by MrEmanuel** [comment]
I would also like to see this fixed. :) 
Here's my error trace if it helps.

```(node:27) Warning: Accessing non-existent property 'count' of module exports inside circular dependency
 emitCircularRequireWarning (internal/modules/cjs/loader.js:824:11)
    at Object.get (internal/modules/cjs/loader.js:838:5)
    at Object.<anonymous> (/user/app/node_modules/migrate-mongo/node_modules/mongodb/lib/operations/db_ops.js:16:42)
    at Module._compile (internal/modules/cjs/loader.js:1201:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1221:10)
    at Module.load (internal/modules/cjs/loader.js:1050:32)
    at Function.Module._load (internal/modules/cjs/loader.js:938:14)
    at Module.require (internal/modules/cjs/loader.js:1090:19)
    at require (internal/modules/cjs/helpers.js:75:18)
    at Object.<anonymous> (/user/app/node_modules/migrate-mongo/node_modules/mongodb/lib/operations/collection_ops.js:5:23)
(node:27) Warning: Accessing non-existent property 'findOne' of module exports inside circular dependency
    at emitCircularRequireWarning (internal/modules/cjs/loader.js:824:11)
    at Object.get (internal/modules/cjs/loader.js:838:5)
    at Object.<anonymous> (/user/app/node_modules/migrate-mongo/node_modules/mongodb/lib/operations/db_ops.js:17:44)
    at Module._compile (internal/modules/cjs/loader.js:1201:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1221:10)
    at Module.load (internal/modules/cjs/loader.js:1050:32)
    at Function.Module._load (internal/modules/cjs/loader.js:938:14)
    at Module.require (internal/modules/cjs/loader.js:1090:19)
    at require (internal/modules/cjs/helpers.js:75:18)
    at Object.<anonymous> (/user/app/node_modules/migrate-mongo/node_modules/mongodb/lib/operations/collection_ops.js:5:23)
(node:27) Warning: Accessing non-existent property 'remove' of module exports inside circular dependency
    at emitCircularRequireWarning (internal/modules/cjs/loader.js:824:11)
    at Object.get (internal/modules/cjs/loader.js:838:5)
    at Object.<anonymous> (/user/app/node_modules/migrate-mongo/node_modules/mongodb/lib/operations/db_ops.js:18:43)
    at Module._compile (internal/modules/cjs/loader.js:1201:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1221:10)
    at Module.load (internal/modules/cjs/loader.js:1050:32)
    at Function.Module._load (internal/modules/cjs/loader.js:938:14)
    at Module.require (internal/modules/cjs/loader.js:1090:19)
    at require (internal/modules/cjs/helpers.js:75:18)
    at Object.<anonymous> (/user/app/node_modules/migrate-mongo/node_modules/mongodb/lib/operations/collection_ops.js:5:23)
(node:27) Warning: Accessing non-existent property 'updateOne' of module exports inside circular dependency
    at emitCircularRequireWarning (internal/modules/cjs/loader.js:824:11)
    at Object.get (internal/modules/cjs/loader.js:838:5)
    at Object.<anonymous> (/user/app/node_modules/migrate-mongo/node_modules/mongodb/lib/operations/db_ops.js:19:46)
    at Module._compile (internal/modules/cjs/loader.js:1201:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1221:10)
    at Module.load (internal/modules/cjs/loader.js:1050:32)
    at Function.Module._load (internal/modules/cjs/loader.js:938:14)
    at Module.require (internal/modules/cjs/loader.js:1090:19)
    at require (internal/modules/cjs/helpers.js:75:18)
    at Object.<anonymous> (/user/app/node_modules/migrate-mongo/node_modules/mongodb/lib/operations/collection_ops.js:5:23)```

**Comment by sechel** [comment]
Same warning here, appreciate a fix 👍

**Comment by sechel** [comment]
A small workaround for this is to overwrite the dependency, for instance when you use yarn in the `resolution` section of `package.json`:
```
  "resolutions": {
    "mongodb": "^3.5.9"
  }
```

**Comment by seppevs** [comment]
I have just published a new version: v7.2.2 of migrate-mongo where I've upgraded all dependencies, inclusing mongodb. 
This issue should be resolved. Can someone verify this?

**Comment by sechel** [comment]
You could also use the `peerDependency` feature since most users will require `mongodb` anyways.

**Comment by StarpTech** [comment]
@seppevs why do you use fixed version instead `3.5.x`?

**Comment by seppevs** [comment]
@StarpTech no reason, so I might switch to that in a later version. For now, I will close this issue because it is resolved.
